code/datums/personal_statistics.dm

```diff
@@ -0,0 +1,565 @@
+/*
+This file is where all variables and functions related to personal lists are defined
+I've done my best to make it as organized for future additions and changes, but yes it is a lot
+
+The personal_statistics_list serves as a large directory for clients and their version of /datum/personal_statistics
+In /datum/personal_statistics is where all the data is stored, manipulated by various procs throughout the code
+
+The most basic way to add to something is like this:
+	var/datum/personal_statistics/personal_statistics = GLOB.personal_statistics_list[client]
+	personal_statistics.stat_you_want_changed++
```

Comment:
you realize clients get destroyed ALL the time right?
thats why I told you to use the ckey, since byond garauntees that it's unique

Reply:
I did not know that. Will change the system to use ckeys. And yes it was working properly with clients.